docs/contribution-guidelines.md

```diff
@@ -0,0 +1,20 @@
+## Contribution Guidelines
+
+### 1 - Fork the project.
+
+### 2 - Create a branch for your feature.
+
+The branch needs to be off of `master`. Name the branch with your first name pre-pended and after the bar `/`, write a [semantic term](https://programadriano.medium.com/dica-r%C3%A1pida-commits-sem%C3%A2nticos-e0ca2139badd#:~:text=Chore%3A%20Utilizamos%20o%20chore%20quando,como%3A%20adicionar%20algo%20dentro%20do%20.) to describe your modifications: \
+ `git checkout -b name/feat-my-feature`.
+
+### 3 - Commit your changes. Always do [semantics commits](https://programadriano.medium.com/dica-r%C3%A1pida-commits-sem%C3%A2nticos-e0ca2139badd#:~:text=Chore%3A%20Utilizamos%20o%20chore%20quando,como%3A%20adicionar%20algo%20dentro%20do%20.):
```

Comment:
Regarding links, I personally like to [move them to the bottom][link], so we don't disrupt reader's experience

```suggestion
The branch needs to be off of `master`. Name the branch with your first name pre-pended and after the bar `/`, write a [semantic term][1] to describe your modifications:
 `git checkout -b name/feat-my-feature`.

### 3 - Commit your changes. Always do [semantics commits][1]:
```

```md
// at the end of the file

[1]: https://programadriano.medium.com/dica-r%C3%A1pida-commits-sem%C3%A2nticos-e0ca2139badd#:~:text=Chore%3A%20Utilizamos%20o%20chore%20quando,como%3A%20adicionar%20algo%20dentro%20do%20.
```

[link]: https://www.markdownguide.org/basic-syntax/#formatting-the-first-part-of-the-link

Reply:
Great! Thank you for the suggestion, I will update the documentation following this format.